spec/features/merchants/new_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe 'New Merchant Page' do
+  it 'creates a new merchant and returns to Merchant Index page' do
+    visit '/merchants/new'
+    expect(current_path).to eq('/merchants/new')
+
+    fill_in :name, with: 'New Merchant'
+    fill_in :address, with: '135 Main'
+    fill_in :city, with: 'Denver'
+    fill_in :state, with: 'Colorado'
+    fill_in :zip, with: '80023'
+
+    click_on 'Create Merchant'
+
+    expect(current_path).to eq('/merchants')
+    expect(page).to have_content('New Merchant')
```

Comment:
Use a different name than "New Merchant" since the link to the new merchant form is also labeled "New Merchant". This could create a false positive where your test is passing even if you didn't successfully create the new merchant